### ISSUE TITLE
Fix Frontend Failing Test: paddle - loss.paddle.nn.functional.margin_ranking_loss

### ISSUE THREAD

**Issue by ivy-leaves** [unifyai/ivy#27710]
### Test Details
- **Function**: `paddle.nn.functional.margin_ranking_loss`
- **Library**: paddle
- **Submodule**: loss

### Test Status
- **Status**: Fail
- **Workflow**: [Link to Workflow](https://github.com/unifyai/ivy/actions/runs/7041659277/job/19164555702)
- **Test Path**: `ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_loss.py::test_paddle_margin_ranking_loss`
